modules/src/applications/transfer/relay/on_recv_packet.rs

```diff
@@ -0,0 +1,86 @@
+use crate::applications::transfer::context::Ics20Context;
+use crate::applications::transfer::error::Error as Ics20Error;
+use crate::applications::transfer::events::DenomTraceEvent;
+use crate::applications::transfer::packet::PacketData;
+use crate::applications::transfer::{IbcCoin, Source, TracePrefix};
+use crate::core::ics04_channel::packet::Packet;
+use crate::core::ics26_routing::context::{ModuleOutputBuilder, WriteFn};
+use crate::prelude::*;
+
+pub fn process_recv_packet<Ctx: 'static + Ics20Context>(
+    ctx: &Ctx,
+    output: &mut ModuleOutputBuilder,
+    packet: &Packet,
+    data: PacketData,
+) -> Result<Box<WriteFn>, Ics20Error> {
+    if !ctx.is_receive_enabled() {
+        return Err(Ics20Error::receive_disabled());
+    }
+
+    let receiver = data
```

Comment:
Can we name this `receiver_account`? "Receiver" and "Sender" can stand for a chain or an account, so it would help readability to make explicit which we're referring to.

Reply:
d6ec510bd7e5583b3739ff094d6fd3ca2194c445